### DISCUSSION TITLE
Super stacked crafting

### DISCUSSION THREAD

**Discussion by mtjohnson47** [P3pp3rF1y/SophisticatedBackpacks#160] [Feature / Idea Archive]
I suggest adding the ability to pick up super sized stacks and place them in the crafting table/stonecutter to craft with.  You can also add support for the furnace but that's pointless since it only crafts 1 at a time anyway.  Clicking a supersized stack in your inventory would only place 64, and leave the remainder in your hand.  Clicking outside the windows would toss a bunch of 64 stacks.

**Comment by P3pp3rF1y** [comment]
Implementing this would require a lot of changes in all kinds of places and it may not be possible at all because crafting recipes are done by mods who don't need to count on bigger stacks in crafting grid so they would likely just delete anything over 64. And really if you need to craft that much backpack shouldn't be the place but rather some autocrafting setup from another mod.
